### PR TITLE
AM-6265: Update the gateway-certs-generator README.md file

### PR DESCRIPTION
**Pull request by uma-kt**
- Rephrased
- Fixed typos/errors
- Fixed hyperlinks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
-# gateway-certs-generator
+# Certificate Generator for Slice Gateway
```
Comment (veenamj-avesha) [inline README.md:1]: I don't know if we can change the title of the ReadMe. It's based on the repo name. 

@narmidm  - can you confirm?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 
-This is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to easy-rsa (typically bundled with OpenVPN).
+The `gateway-certs-generator` is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to `easy-rsa` (typically bundled with OpenVPN).
 
-easy-rsa is a CLI utility to build and manage a PKI CA. In layman's terms, this means to create a root certificate authority, and request and sign certificates including intermediate CAs and certificate revocation lists (CRL).
+To create and maintain a PKI CA, use the CLI tool `easy-rsa`. Using the tool create a root certificate authority, and request and sign certificates including intermediate CAs and certificate revocation lists (CRL).
```
Comment (veenamj-avesha) [inline README.md:5]: To create and manage  a PKI CA - would sound better?

File: README.md [hunk 3]
```diff
@@ -1,21 +1,26 @@
-# gateway-certs-generator
+# Certificate Generator for Slice Gateway
 
-This is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to easy-rsa (typically bundled with OpenVPN).
+The `gateway-certs-generator` is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to `easy-rsa` (typically bundled with OpenVPN).
```
Comment (veenamj-avesha) [inline README.md:3]: change it to It as it's the first word of a new sentence.

File: README.md (hunk 2 above)
Comment (veenamj-avesha) [reply README.md:5]: The previous sentence ends with `easy-rsa`. 

So, easy-rsa is a CLI tool/utility to build and manage a PKI CA would be better rephrasing - as we explain what `easy-rsa` is...

Comment (veenamj-avesha) [inline README.md:5]: A comma should be there after `Using the tool`

File: README.md [hunk 4]
```diff
@@ -68,7 +73,7 @@ docker exec -it kind-control-plane crictl images
 ```
 
 ### Deploying in a Cluster
-1. Create chart values file `yourvaluesfile.yaml`. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) on how to adjust this and update the `kubeslice-controller` image to the local build image.
+1. Create the chart values `yourvaluesfile.yaml` file. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) on how to adjust this and update the `kubeslice-controller` image to the local build image.
```
Comment (veenamj-avesha) [inline README.md:76]: The previous usage was correct - 

Add a comma after file 

Rephrase it as `Create a chart values file called yourvaluesfile.yaml. `

File: README.md [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
+
+To create certificates, the controller needs the `gateway-cert-generator` image. Therefore, we create the image and use the image version in the controller values file.
 
-* Docker installed and running in your local machine
-* A running [`kind`](https://kind.sigs.k8s.io/)  cluster
+### Prerequisites
+Before you begin, make sure the following prerequisites are met:
+* Docker is installed and running on your local machine.
+* A running [`kind`](https://kind.sigs.k8s.io/) cluster.
 * [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed and configured
```
Comment (veenamj-avesha) [inline README.md:22]: kubectl is installed and configured ( add a period at the end of the sentence) - as other items are full sentences.

File: README.md [hunk 6]
```diff
@@ -1,21 +1,26 @@
...
-* Follow the getting started from above, to install [`kubeslice-controller`](https://github.com/kubeslice/kubeslice-controller) and [`worker-operator`](https://github.com/kubeslice/worker-operator)
+* You have prepared the environment for the installation of [`kubeslice-controller`](https://github.com/kubeslice/kubeslice-controller) on the controller cluster and [`worker-operator`](https://github.com/kubeslice/worker-operator) on the worker cluster. For more information, see [Prerequisites](https://kubeslice.io/documentation/open-source/0.5.0/getting-started-with-cloud-clusters/prerequisites/).
```
Comment (veenamj-avesha) [inline README.md:23]: `for installing` or `to install` 

use verb forms over noun forms for better readability

File: README.md (hunk 3 above)
Comment (uma-kt) [reply README.md:3]: updated

File: README.md (hunk 2 above)
Comment (uma-kt) [reply README.md:5]: done

Comment (uma-kt) [reply README.md:5]: done

File: README.md (hunk 6 above)
Comment (uma-kt) [reply README.md:23]: updated

File: README.md (hunk 5 above)
Comment (uma-kt) [reply README.md:22]: done

File: README.md (hunk 4 above)
Comment (uma-kt) [reply README.md:76]: updated
